bubblegum/program/src/lib.rs

```diff
@@ -119,6 +122,30 @@ pub struct CreatorVerification<'info> {
     pub merkle_slab: UncheckedAccount<'info>,
 }
 
+#[derive(Accounts)]
+pub struct CollectionVerification<'info> {
+    #[account(
+        seeds = [merkle_slab.key().as_ref()],
+        bump,
+    )]
+    pub authority: Account<'info, TreeConfig>,
+    /// CHECK: This account is checked in the instruction
+    pub owner: UncheckedAccount<'info>,
+    /// CHECK: This account is checked in the instruction
+    pub delegate: UncheckedAccount<'info>,
+    pub collection_authority: Signer<'info>,
+    /// CHECK: This account is checked in the instruction
+    pub collection_mint: UncheckedAccount<'info>,
+    pub collection_metadata: Box<Account<'info, TokenMetadata>>,
```

Comment:
Yes I found that out (that it needed to be mut) in my local copy where I'm working on sized collection support.

Although rather than CPI to token metadata, I think I may be able to just increment it and reserialize it in the bubblegum contract, because `AccountSerialize` and `AccountDeserialize` were implemented for the struct `TokenMetadata(mpl_token_metadata::state::Metadata)` in metaplex_anchor.rs.

Reply:
Updated in latest version.

Reply:
Actually I take that back I think you were right and it will have to be a CPI to token-metadata as that program is the account owner.

Reply:
Converting this to a CPI now

Reply:
Converted to use CPI in latest version.